packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -479,6 +479,8 @@ class _CupertinoScrollbarState extends State<CupertinoScrollbar> with TickerProv
   }
 
   bool _handleScrollNotification(ScrollNotification notification) {
+    if (notification.depth != 0)
```

Comment:
We should make this configurable, some folks may expect their nested scrollbars to show up. This would break them.
I think this is one such case https://github.com/flutter/flutter/issues/70380